azure-pipelines.yml

```
@@ -226,13 +227,7 @@ jobs:
         filePath: eng/build.ps1
         arguments: -configuration Debug -prepareMachine -ci -restore -binaryLog
 
-    - task: PowerShell@2 
-      displayName: Build
-      inputs:
-        filePath: eng/build.ps1
-        arguments: -configuration Debug -prepareMachine -ci -build -bootstrap -publish -binaryLog -skipDocumentation
-
-    - script: .\artifacts\bin\BuildValidator\Debug\net472\BuildValidator.exe --assembliesPath .\artifacts\obj\Microsoft.CodeAnalysis --debugPath .\artifacts\BuildValidator --sourcePath .
+    - powershell: .\eng\test-rebuild.ps1
```

```suggestion
    - powershell: .\eng\test-rebuild.ps1 -ci
```

The script still has a `-ci` argument